querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java

```diff
@@ -33,6 +33,9 @@
 import com.querydsl.sql.codegen.DefaultNamingStrategy;
 import com.querydsl.sql.codegen.MetaDataExporter;
 import com.querydsl.sql.codegen.NamingStrategy;
+import com.querydsl.sql.codegen.support.NumericMapping;
+import com.querydsl.sql.codegen.support.RenameMapping;
+import com.querydsl.sql.codegen.support.TypeMapping;
```

Comment:
Are these imports necessary?
They seem to be the only thing that changed in this file.
Or am I missing something?